src/book.py

```diff
@@ -363,6 +363,123 @@ def _read_kraken_ledgers_old(self, file_path: Path) -> None:
+        """
+
+        platform = "bitpanda_pro"
+        with open(file_path, encoding="utf8") as f:
+            reader = csv.reader(f)
+
+            # should be true in any case, since this has been dispatched here
+            # so simply skip
+            line = next(reader)
+            # assert line[0] == \
+            # "Disclaimer: ..."
+
+            # for transactions, it's currently written "id" (small)
+            line = next(reader)
+            if line[0].startswith("Account id :"):
+                log.warning(
+                    f"{file_path} looks like a Bitpanda transaction file. Skipping."
+                )
+                return
+
+            assert line[0].startswith("Account ID:")
+            line = next(reader)
+            # empty line - still keep this check in case Bitpanda changes the
+            # transaction file to match the trade header (casing)
+            if not line:
+                log.warning(
+                    f"{file_path} looks like a Bitpanda transaction file. Skipping."
+                )
+                return
+            elif line[0] != "Bitpanda Pro trade history":
+                log.warning(
+                    f"{file_path} doesn't look like a Bitpanda trade file. Skipping."
+                )
+                return
+
+            line = next(reader)
+            assert line == [
+                "Order ID",
+                "Trade ID",
+                "Type",
+                "Market",
+                "Amount",
+                "Amount Currency",
+                "Price",
+                "Price Currency",
+                "Fee",
+                "Fee Currency",
+                "Time (UTC)",
+            ]
+
+            for (
+                _order_id,
+                _trace_id,
+                operation,
+                trade_pair,
+                amount,
+                amount_currency,
+                _price,
+                price_currency,
+                fee,
+                fee_currency,
+                _utc_time,
+            ) in reader:
+                row = reader.line_num
+                # make RFC3339 timestamp ISO 8601 parseable
+                if _utc_time[-1] == "Z":
+                    _utc_time = _utc_time[:-1] + "+00:00"
+
+                utc_time = datetime.datetime.fromisoformat(_utc_time)
+                # not needed, already taken care of
+                # utc_time = utc_time.replace(tzinfo=datetime.timezone.utc)
+
+                # trade pair is of form e.g. BTC_EUR
+                assert [amount_currency, price_currency] == trade_pair.split("_")
+                # there shouldn't be any other types
+                assert operation == "BUY" or operation == "SELL"
+
+                coin = amount_currency
+                # always positive, regardless whether BUY or SELL
+                change = misc.force_decimal(amount)
+                # just to make sure it doesn't get changed some day
```

Comment:
This comment (line) is not needed. It doesn't add any information. Asserts are here for exactly that.